src/main/java/com/faforever/client/moderator/ModeratorService.java

```diff
@@ -0,0 +1,51 @@
+package com.faforever.client.moderator;
+
+import com.faforever.client.api.FafApiAccessor;
+import com.faforever.client.api.dto.LegacyAccessLevel;
+import com.faforever.client.api.dto.LobbyGroup;
+import com.faforever.client.remote.FafService;
+import com.faforever.client.remote.domain.PeriodType;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+import java.util.concurrent.CompletableFuture;
+
+@Service
+public class ModeratorService {
+  private static CompletableFuture isModerator;
+  private final FafService fafService;
+  private final FafApiAccessor fafApiAccessor;
+
+  public ModeratorService(FafService fafService, FafApiAccessor fafApiAccessor) {
+    this.fafService = fafService;
+    this.fafApiAccessor = fafApiAccessor;
+  }
+
+
+  public CompletableFuture<Boolean> isModerator() {
```

Comment:
cache this... this is called from every chat user item i dont wanan have server crashing if somebody scolls user list

Reply:
why?
The completable future is initialized with the first call and then the call to the API will never ever trigger again.

Reply:
true did not see that